app/components/AuthenticatedLayout.tsx

```diff
@@ -39,6 +40,18 @@ const AuthenticatedLayout: React.FC = ({ children }) => {
   const location = useLocation();
   const can = usePolicy(ui.activeCollectionId);
   const { user, team } = auth;
+  const previousPath = usePrevious(location.pathname);
```

Comment:
I think you might have misunderstood the intent of the feature. The idea is that when navigating to `http://company.getoutline.com` it would remember which document or collection you were previously viewing and then redirect to that.

`usePrevious` is not going to work for this, it just gives you the value of the last render loop. We'll need to store the last viewed pathname as you navigate around and then use that here. Would potentially use the `usePersistedState` hook for this.

Reply:
Thanks!

_Note to self_ :point_down: 

There are two parts of the problem,
1. Persist last visited document or collection path.
2. Redirect to path in (1) when matched to `/`(make sure there's no UI flash in this step).